scripts/ci/gitlab/nextest.toml

```diff
@@ -108,17 +108,25 @@ store-success-output = false
 store-failure-output = true
 
 # This profile is activated if MIRI_SYSROOT is set.
-[profile.default-miri]
+# [profile.default-miri]
 # Miri tests take up a lot of memory, so only run 1 test at a time by default.
-test-threads = 1
-
-# Mutual exclusion of tests with `cargo build` invocation as a lock to avoid multiple
-# simultaneous invocations clobbering each other.
-[test-groups]
-serial-integration = { max-threads = 1 }
 
 # Running UI tests sequentially
 # More info can be found here: https://github.com/paritytech/ci_cd/issues/754
 [[profile.default.overrides]]
-filter = 'test(/(^ui$|_ui|ui_)/)'
-test-group = 'serial-integration'
+# To list all the tests, use this command:
+# cargo nextest list --workspace --locked --release --features=runtime-benchmarks
+#
+# To check if you added your test to a filter and group correctly, use this command:
+# cargo nextest show-config test-groups --config-file scripts/ci/gitlab/nextest.toml --features runtime-benchmarks --workspace --locked --release --manifest-path ./bin/node/cli/Cargo.toml
+# For the non-default [[profile.default.overrides]] i.e. [[profile.local.overrides]] add `--profile local` to the above command
+filter = "test(follow_report_multiple_pruned_block) | test (follow_forks_pruned_block) | test(can_sync_small_non_best_forks) | test(telemetry_works) | test(running_the_node_works_and_can_be_interrupted) | test(notifications_back_pressure)"
+# Flaky tests are executed exclusively
+test-group = 'flaky'
```

Comment:
These name is misleading. Flaky tests are differ from mutual exclusive ones.

Reply:
this group is sorta TODO by design, refers to the tests that are noted to be flaky and require fixing, but not yet.

Reply:
Initialy group was created for test which require single thread run. They are not flaky. We know the nature and behavior of them is consistent.